.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
+install:
+  - pip install pandas
+  - pip install codecov
+# command to run tests
+script:
+  - coverage run -m unittest discover
+after_success:
+  - bash <(curl -s https://codecov.io/bash) -t $CODECOV_TOKEN
```

Comment:
You don't actually need to use `-t $CODECOV_TOKEN` here, just leave this off, the script will look for CODECOV_TOKEN.

Otherwise, LGTM!

Reply:
Oh! My apologies. Just pushed a fix to remove that.